### DISCUSSION TITLE
Antela Smart 9W RGB & 2700K-6500K

### DISCUSSION THREAD

**Discussion by davidebeatrici** [arendst/Tasmota#15814] [Q&A] [answered]
Unknown chip, can anyone identify it?

<details>
<summary>Photos</summary>

![](https://user-images.githubusercontent.com/5897523/177239911-c0b278d5-133a-456c-9700-074396f9c982.png)
![](https://user-images.githubusercontent.com/5897523/174232714-d12c67ef-71fe-4ad4-926c-ade2527d9e30.png)
![](https://user-images.githubusercontent.com/5897523/174232720-22e3faf6-48e9-4ce0-b724-0022de7e7b7f.png)
![](https://user-images.githubusercontent.com/5897523/174232722-ff82eb36-fe84-4e77-90ac-f62fdad5c098.png)
![](https://user-images.githubusercontent.com/5897523/174232727-eb92b0ea-e795-445f-9af8-d3d86f46d76c.png)
</details>

The QR code evaluates to `ZL1168401M05524`.

---

[Amazon.it](https://www.amazon.it/gp/product/B08R8PP7Q9)

[Official website](https://www.antelaglobal.com/products/lampadina-alexa-inteligente-wifi-led-bulb-e27-dimmerabile-lampadina-antela-smart-9w-806ml-80w-equivalente-rgb-2700k-6500k-bianco-freddo-caldo-compatibile-con-alexa-google-home-2-pezzi-%E7%9A%84%E5%89%AF%E6%9C%AC)

On a separate note, a different [product](https://www.antelaglobal.com/products/antela-led-bulb-10w-1000lm) appears to be free on the website:

![](https://user-images.githubusercontent.com/5897523/174231470-67edf148-374a-4fb2-8b1c-5a8a2c7d1b6c.png)

Probably a mistake.

**Comment by sfromis** [comment, marked as answer]
Your last image has glaring reflexes in the very most interesting spot, center above the metal can. That's where many modules have the type printed.

Check the MAC address in pairing mode, and look it up on the Internet. Only Tasmota-compatible if Espressif. Form factor appears "similar" to [TYWE2L](https://tasmota.github.io/docs/Pinouts/#tywe2l), but that is no strong indicator of the MCU inside still being ESP8285.

Still, it is a plausible link for pinout, including pads for flashing on the back side. Would take desoldering the module if it actually is TYWE2L / WE2L.

**Reply by davidebeatrici** [reply]
I think you're right!

![](https://user-images.githubusercontent.com/5897523/174254410-dbbe279a-c354-417e-b78a-5b3f15804b0a.png)

I will check the MAC address.

**Reply by sfromis** [reply]
Forget it. [CB2L](https://developer.tuya.com/en/docs/iot/cb2l-module-datasheet?id=Kai2eku1m3pyl) has BK7231N instead of ESP8285. Tasmota cannot work on that chip. You could replace the module. Maybe try https://github.com/openshwprojects

**Reply by sfromis** [reply]
This module should be pin compatible for replacement, if you ignore the extra pins on the sides.
https://templates.blakadder.com/DMP-L1.html

**Reply by davidebeatrici** [reply]
Thanks! I will try to flash Tasmota for BK7231N.

The CB2L is only soldered on the 6 pins at the bottom, right?

**Reply by sfromis** [reply]
Tasmota is not available for the BK7231, but that other project may be usable. I'd not expect the CB2L to be soldered with the test pads likely available on the back side, and I'd not expect glue either, but I have no real knowledge. My guess would be that only the visible pins at the end is soldered.

**Reply by davidebeatrici** [reply]
Oh, sorry, I just realized https://github.com/openshwprojects/OpenBK7231T_App is a different firmware and not a port of Tasmota.

I'll keep you updated.

**Reply by davidebeatrici** [reply]
Unfortunately I discovered that pins are actually soldered on the underside as well in the worst way possible: by accidentally ripping the pads off thinking it was glue... I should've used hot air instead of solder wick and iron.

I didn't stop there though. I soldered the required wires for UART and attempted to access the serial interface, until I realized the TX1 pad was gone with the Dupont jumper just hanging there. Either the pads are too weak or it's just an unlucky operation overall.

Is the DMP-L1 the only possible replacement in order to run Tasmota?

**Reply by sfromis** [reply]
Oops :disappointed: 

https://templates.blakadder.com/DT-ESP-C05.html appears to be similarly pin compatible at the visible end, with ESP32-C3 (also supported by Tasmota) instead of ESP8266, but that's basically just an improvement with hardware PWM and more capacity for extra features of Tasmota.

In principle any module can be used with a birds nest of wires, but that can get a mess inside a bulb.

**Reply by davidebeatrici** [reply]
I agree, especially with the AC circuit being so close to the DC one.

https://www.aliexpress.com/item/1005003690845216.html
https://www.aliexpress.com/item/1005003691435557.html

Considering the tiny price difference, I'll go with the DT-ESP-C05.

But first, I have to make sure the ripped pads did not lead anywhere.

**Reply by davidebeatrici** [reply]
Looks like:

1. I'll have to rebuild 3 tracks.
2. The chip was indeed glued!

<details>
<summary>Photos</summary>

![](https://user-images.githubusercontent.com/5897523/174939382-2f7a6a0e-33d4-4f43-ad27-8ec4e8afa4bd.png)
![](https://user-images.githubusercontent.com/5897523/174939379-c83e246e-211c-4c41-a7ef-f0ed760c5e5f.png)
</details>

**Reply by davidebeatrici** [reply]
@sfromis The pad that P24 was attached to is completely gone, but I don't see any traces leading to it. Was it perhaps only for rigidity?

**Reply by sfromis** [reply]
Depends on how many pins are needed for control. PWM takes up to 5 for RGBCW, but seeing labels DATA and CLK makes it look like those two are going to a LED controller chip (something with 2135?) on the disc. Does not really look like more than a single-layer PCB, or there just might be a "hidden" layer sandwiched. Anyway, soldering pads for rigidity can happen, but then it looks a bit odd with one pad not soldered.

**Reply by davidebeatrici** [reply]
That's what I thought. I put a light under the PCB and I could see all traces, definitely single-layer.

The reason for the single pad not soldered is the trace leading to R2 and R3.

The two chips on the disc are `BP5218` and `BP5758`.

#15713

**Reply by sfromis** [reply]
By now, I suppose that you can see why we sometimes suggest Zigbee bulbs for local control with Tasmota :wink:

**Reply by davidebeatrici** [reply]
Wouldn't they need a hub though?

**Reply by sfromis** [reply]
Sure, the coordinator is where Tasmota gets involved, as it cannot reside on the bulb. Of course, Tasmota is not the only local-only coordinator setup available.

**Reply by davidebeatrici** [reply]
In my case I would prefer all devices to be independent and in my local network, with Tasmota of course.

So far I've been lucky with bulbs, they all had ESP chips out of the box (with the exception of the one(s) in this discussion).

But yeah, I can see the advantage in setting up a Tasmota coordinator and connecting any Zigbee devices without compromising privacy and security.

**Reply by sfromis** [reply]
And most people do not enjoy taking bulbs apart :wink:

**Reply by davidebeatrici** [reply]
Indeed, especially when the AC wires are soldered to the bulb's base...

**Reply by nizar0x1f** [reply]
I found the same PCB in your pictures inside the [Imou B5 Wi-Fi Bulb](https://www.imou.com/en/product/detail/b5). Sadly, it is useless.

**Reply by davidebeatrici** [reply]
Too bad...
